pontoon/base/templates/translate.html

```diff
@@ -167,7 +167,7 @@ <h3 class="no-match"><div class="fa fa-exclamation-circle"></div>No results</h3>
       <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
     </div>
     <div class="batch-bar topbar clearfix">
-      <a class="select-all" href="#select-all" title="Select All Strings (Ctrl + Alt + A)">
+      <a class="select-all" href="#select-all" title="Select All Strings (Ctrl + Shift + A)">
```

Comment:
A total nit, but we could create a single jinja2 module that would contain all description of shortcuts "ctrl + shift + ..." in one place.